### ISSUE TITLE
After Upgrade to 7.x.x the transactions dashboards based on httpResponseCode doesn't show any data 

### ISSUE THREAD

**Issue by A7med-Ragheb** [newrelic/node-newrelic#762]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT share sensitive information, whether personal, proprietary, or otherwise! )

After Upgrade to 7.x.x from 6.x.x all the transactions dashboards based on **httpResponseCode** doesn't show any data any more. 

## Expected Behavior

[NOTE]: # ( Tell us what you expected to happen. )

Transaction Dashboard based on   **httpResponseCode**  should display data.

![Screenshot 2021-05-21 at 10 04 14](https://user-images.githubusercontent.com/81766970/119104251-57cf8300-ba1c-11eb-9671-5437a724ecc3.png)

[NOTE]: # ( Provide any other relevant log data. )
[TIP]:  # ( Scrub logs and diagnostic information for sensitive information )

## Steps to Reproduce

[NOTE]: # ( Please be as specific as possible. )
[TIP]:  # ( Link a sample application that demonstrates the issue. )

* Use version 7.x
* Run a query transaction `SELECT httpResponseCode FROM Transaction WHERE appName like 'YOUR_APP'`
## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )

* Browser name and version: Chrome latest
* Node version: 14.x
* Operating System and version: MAC OS

**Comment by michaelgoin** [comment]
Hi @A7med-Ragheb,

This is expected. That was one of the breaking changes called out in the version 7.0 release notes.

`* **BREAKING** Removed deprecated httpResponseCode, response.status and
  httpResponseMessage http response attributes`.
  
  That attribute had been previously deprecated in version v6.4.2. You'll want to update your dashboards to use the following which were introduced in v6.4.2:
  - http.statusCode
  - http.statusText (if using httpResponseMessage anywhere)

Sorry for the inconvenience!

**Comment by A7med-Ragheb** [comment]
@michaelgoin Thanks for the update, I think this bug can be closed then.

**Comment by BrightStoresInc** [comment]
hey @michaelgoin - makes sense, I'm still not seeing any data on 7.0:

`SELECT 
filter(count(*), WHERE http.statusCode LIKE '2%') as '200', 
filter(count(*), WHERE http.statusCode LIKE '3%') as '300',
filter(count(*), WHERE http.statusCode LIKE '4%') as '400',
filter(count(*), WHERE http.statusCode LIKE '5%') as '500'
FROM Transaction WHERE appName='<app name>' SINCE 15 minutes ago`

Maybe an issue with the syntax?

**Comment by michaelgoin** [comment]
@BrightStoresInc hmm, I'm not sure why that isn't working.

I just ran that for my own app, swapping out my app name, and I saw everything grouped into a table under the appropriate status codes once the data landed on the server. This is with v7.4.0.

I would first verify you are seeing the Transaction event data you'd expect for your application you just deployed.

`FROM Transaction SELECT * WHERE appName = '<app name>`

And check to see those new attributes. They render slightly differently in that table view: 'http.status Code' and 'http.status Text' is how they render.

If you don't see events at all... then something has gone wrong and we'll want to verify your app is deployed successfully, that the agent installed successfully and that the agent is running / didn't log that it had issues.

If we see transaction events but not the new attributes, I'd double check the old attributes are not there and the upgrade succeeded.

If none of the attributes are there for http status code (old or new) but you are getting data, I'd double check your attribute filtering rules.

And if all of that checks out / there's no obvious cause, you'll probably want to go through the official support channels so we can deep-dive into your application, configuration, etc. and try to help narrow-down the issue.
